docker/Dockerfile

```diff
@@ -0,0 +1,130 @@
+FROM ubuntu:16.04
+
+MAINTAINER Daniel Baptista Dias <danielbpdias@gmail.com>
+
+ENV GENSIM_REPOSITORY https://github.com/RaRe-Technologies/gensim/archive
+ENV GENSIM_VERSION master
+
+# Installs python, pip and setup tools (with fixed versions)
+RUN apt-get update \
+    && apt-get install -y \
+    ant=1.9.6-1ubuntu1 \
```

Comment:
Another thing: In my previous PR I forgot to add a modification in [dockerfile](https://github.com/danielbdias/gensim/blob/issue-497/docker/Dockerfile) that fix some dependencies for git dependencies to avoid this problems.

I was following the docker [contribution guidelines](https://github.com/docker-library/official-images) to turn this image into an official one in the future.